### DISCUSSION TITLE
Chargement de valeurs pour les villes ou provinces

### DISCUSSION THREAD

**Discussion by MpmMess** [MTESSDev/FRW#9] [Annonces]
Est-ce que FRW offre des listes de valeurs, comme pour exemple, les villes d'une province ou bien les provinces tout simplement ?

**Comment by j0ber** [comment]
Non, FRW ne fourni pas de liste de valeurs. 
Les valeurs possibles doivent être saisies dans le fichier de config de votre formulaire.

Nous allons bientôt mettre en place la possibilité de recevoir en entrée (pré-remplissage) des domaines de valeurs. Cela pourrait être utilisé dans ce cas.

Voir : https://github.com/MTESSDev/FRW/issues/19

**Comment by drogu02** [comment]
Je me suis fait demander si ça l'existait un champ de type liste de municipalités ou villes dans FRW. 

Je ne crois pas que ça l'existe. On pourrait utiliser le champ UTD de liste déroulante (multiple ou pas avec recherche).

Évidemment, il faudrait que ce champ soit constamment mis à jour automatiquement, donc provenant des données d'un autre système qui connait les municipalités et villes du Québec.

**Reply by j0ber** [reply]
Bonjour le composant liste déroulante a été livré aujourd'hui.

Cela dit, il faudrait lui passer une liste de toutes les villes du québec... Ce qui ne fait pas vraiment de sens.

Nous avons une story au backlog à propos de pouvoir connecter des champs (ex. la liste déroulante) à un api afin d'obtenir le domaine de valeur et/ou filtrer celui-ci en fonction de ce qui est saisi.

Donc pour le moment ce n'est pas possible, mais ça risque de l'être dans le futur. Il faudra aussi se trouver un api fiable qui fourni cette liste de villes du Québec. Au ministère nous avons RIR, mais tout est en majuscule, ce qui n'est pas l'idéal.

**Reply by drogu02** [reply]
Je vais m'informer au MAMH!  (Pour ce qui est des municipalités)

**Reply by drogu02** [reply]
Il y a ça qui existe pour les municipalités

https://www.donneesquebec.ca/recherche/dataset/repertoire-des-municipalites-du-quebec/resource/19385b4e-5503-4330-9e59-f998f5918363
